x/name/keeper/keeper_test.go

```diff
@@ -320,3 +323,31 @@ func (s *KeeperTestSuite) TestSecp256r1KeyAlgo() {
 		s.NoError(err)
 	})
 }
+
+func (s *KeeperTestSuite) TestAuthority() {
+	require.EqualValues(s.T(), s.app.NameKeeper.GetAuthority(), "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn")
+}
+
+func (s *KeeperTestSuite) TestCreateRootName() {
+	s.msgSrvr = namekeeper.NewMsgServerImpl(s.app.NameKeeper)
+	msg := nametypes.MsgCreateRootNameRequest{
+		Record: &nametypes.NameRecord{
+			Name:       "swampmonster",
+			Address:    "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn",
+			Restricted: false,
+		},
+		Authority: "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn",
+	}
+
+	s.Run("create valid root name", func() {
+		_, err := s.msgSrvr.CreateRootName(s.ctx, &msg)
+		s.Require().NoError(err)
+	})
+
+	s.Run("invalid authority", func() {
+		msg.Authority = "..."
+		_, err := s.msgSrvr.CreateRootName(s.ctx, &msg)
+		s.Require().Error(err)
+		s.Require().Equal("expected cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn got ...: expected gov account as only signer for proposal message", err.Error())
```

Comment:
For future reference, these two can be combined using `EqualError`. There's also `ErrorContains` which is good to use when multiple things might be involved in an error's final message.

Feel free to resolve this comment.